config/routes.rb

```diff
@@ -71,6 +71,7 @@
         get 'resources'
         get 'templates'
         get 'nics'
+        get 'forget_status'
```

Comment:
I believe this should be
```suggestion
        delete 'forget_status'
```

Reply:
Extending `action_permission` fixed it.  I do think it would make sense for the `get` there to be a `delete` or a `post`, but I'm not quite sure how to make the `clear` link not trigger a `get`.

Reply:
@ianballou: Please find an example here (look for `method`): https://github.com/theforeman/foreman/blob/ae96cb70130968bdcc8b7a9a9af7edc7c6e0d9a8/app/views/key_pairs/index.html.erb#L28